trino-connector/src/main/java/org/apache/gravitino/trino/connector/GravitinoConfig.java

```diff
@@ -69,6 +69,20 @@ public class GravitinoConfig {
           "true",
           false);
 
+  private static final ConfigEntry GRAVITINO_CLOUD_REGION_CODE =
+      new ConfigEntry(
+          "gravitino.cloud.region-code",
+          "The property to specify the region code of the cloud that the catalog is running on.",
+          "",
+          false);
+
+  private static final ConfigEntry GRAVITINO_CATALOG_CONNECTOR_FACTORY_CLASS_NAME =
+      new ConfigEntry(
+          "gravitino.catalog.connector.factory.class.name",
+          "The class name for the special CatalogConnectorFactory.",
```

Comment:
for the special ->  custom

Reply:
Please add the following contents.

The specific class should implement interface `CatalogConnectorFactory`